api/FormData.json

```diff
@@ -52,6 +52,58 @@
           "deprecated": false
         }
       },
+      "FormData": {
+        "__compat": {
+          "description": "<code>FormData()</code> constructor",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FormData/FormData",
+          "support": {
+            "webview_android": {
```

Comment:
Would you mind sorting this down alphabetically?  I admit to poorly adhering to this in previous reviews and PRs myself, but I'm trying to do better